style.css

```diff
@@ -60,3 +60,16 @@
 #sorcery-button .sorcery-run-wrapper code-input {
     grid-area: 1/1;
 }
+
+.sorcery-id {
```

Comment:
Scope these within `#sorcery-button` like the other style rules in order to prevent polluting the rest of the application.